### PR TITLE
#42 prepared nova code for smooth install

### PR DESCRIPTION
**Pull request by Adil9994**

### CODE REVIEW

**Comment by DenTray** [comment]
@Adil9994 as discussed, we no need to install Nova to empty project, just prepare project to installation

**Comment by Goodmain** [comment]
@DenTray I resolved conflicts. Please check again

**Review by DenTray — COMMENTED** [review]
@Goodmain could you please refine whether we should keep migration class and composer change here?

**Comment by Goodmain** [comment]
@DenTray I removed migration and `auth` config file changes related to the `reset_passwords` table.

File: composer.json [hunk 1]
```diff
@@ -53,6 +53,12 @@
       "App\\Tests\\": "tests/"
     }
   },
+  "repositories": [
+    {
+      "type": "composer",
+      "url": "https://nova.laravel.com"
+    }
+  ],
```
Comment (DenTray) [inline composer.json]:
```suggestion
```

**Review by DenTray — CHANGES_REQUESTED** [review]

**Pull request merged**